### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by arthur10031198**
- [DEMO LINK](https://arthur10031198.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://arthur10031198.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
GJ :+1: 
Let's add some improvements/fixes

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Could you read the comments from the previous review carefully and fix them? Also, there is a default outline here 
Screenshot 2023-11-28 at 13 12 44

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Feel free to ask for help in the chat to fix issues here

**Comment by arthur10031198** [comment]
you have changes in the code?

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,23 @@
...
-    <h1>Search bar airbnb</h1>
+    <form class="search-panel" data-qa="big">
+      <label for="bigSearch"></label>
+      <img class="icon" src="images/Search.svg" alt="icon1">
+      <input type="text"
+       id="bigSearch" class="search-bar"
+         data-qa="keypress"
+        placeholder="Try “Los Angeles”">
+    </form>
+    <form class="search-panel-2" data-qa="small" >
```
[the remark this answers is not in the document]
Comment (VitaliyBodnar) [reply src/index.html:22]: Not fixed

File: src/style.css [hunk 2]
```diff
@@ -1 +1,141 @@
-/* add styles here */
+
+/* @font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
```
[the remark this answers is not in the document]
Comment (VitaliyBodnar) [reply src/style.css:9]: Not fixed

File: src/style.css [hunk 3]
```diff
@@ -1 +1,141 @@
...
+.search-bar {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  color: hsl(212, 23%, 31%);
+  line-height: normal;
+  font-style: normal;
+  font-weight: 300;
+  height: 70px;
+  width: 100%;
+  box-sizing: border-box;
```
Comment (VitaliyBodnar) [inline src/style.css:62]: It's better practice to put the box-sizing to this block `*{}` as it this be applied in everywhere

File: src/style.css [hunk 4]
```diff
@@ -1 +1,141 @@
...
+  background: rgb(255, 255, 255);
+  box-shadow: 1px 1px 8px 1px #3d4e611a;
+  display: flex;
+  /* padding-right: 1px; */
+  align-items: center;
+
+}
+
+.search-bar-2 {
+  font-family: "Avenir", sans-serif;
```
Comment (VitaliyBodnar) [inline src/style.css:75]: Font-family is duplicated, so please move it to tag body

File: src/style.css [hunk 5]
```diff
@@ -1 +1,141 @@
...
+  border-radius: 4px;
+  border: 1px solid hsl(210, 25%, 91%);
+  background: hsl(0, 0%, 100%);
+
+}
+
+.search-panel-2 {
+  position: relative;
+  margin-top: 20px;
+
```
Comment (VitaliyBodnar) [inline src/style.css:94]: Redundant spaces
Check other places 
```suggestion
```

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
.

**Pull request closed without merging**